features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -354,4 +381,57 @@ class MessageComposerPresenter @Inject constructor(
                 snackbarDispatcher.post(snackbarMessage)
             }
         }
+
+    @OptIn(FlowPreview::class)
+    private fun CoroutineScope.processComposerSuggestions() = launch {
```

Comment:
Maybe extract this logic in his own file too? 
Also makes sure it's done on a background dispatcher?